chains/saakuru/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+blockExplorers:
+  - apiUrl: https://explorer.saakuru.network/api-docs
+    family: blockscout
+    name: Saakuru Explorer
+    url: https://explorer.saakuru.network
+blocks:
+  confirmations: 1
+  estimateBlockTime: 2
+  reorgPeriod: 0
+chainId: 24734
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Saakuru
+domainId: 24734
```

Comment:
this chain id was also wrong? should be 7225878

Reply:
@paulbalaji sorry for missing these, thanks for catching